extensions/eclipse-che-theia-plugin-remote/src/node/hosted-plugin-remote.ts

```diff
@@ -41,6 +43,11 @@ export class HostedPluginRemote {
      */
     private pluginsMetadata: Map<string, PluginMetadata[]> = new Map<string, PluginMetadata[]>();
 
+    /**
+     * mapping between resource request id (pluginId_resourcePath) and resource query callback.
```

Comment:
Maybe `mapping` should starts from capital letter. But up to you.

Reply:
Other similar comments are written that way, so I decided not to break the style, but will change all of them to capital letter.